socorro/external/es/new_crash_source.py

```diff
@@ -1,103 +0,0 @@
-    )
-
-    def __init__(self, config, name=None, quit_check_callback=None):
-        self.config = config
-        self.es_context = self.config.elasticsearch.elasticsearch_class(
-            config=self.config.elasticsearch
-        )
-
-    def new_crashes(self, date, product, versions):
-        """Return an iterator of crash IDs.
-
-        Confusingly, inside this method we're using an iterator.
-        The reason we first exhaust the iterator BEFORE returning it
-        is because *if* we do this:
-
-            res = helpers.scan(...)
-            for hit in res:
-                yield hit['fields']['crash_id'][0]
-
-        Then, between each of these yields the fetch transform save
-        app will do something with this crash ID. That's cute but the
-        problem is that if you get 1,000 crash IDs out and between each
-        1,000 you have to do a S3 boto get, you have to wait many
-        milliseconds between each and then the scroll connection
-        has to stay open too long.
-
-        Instead, we get all the crash IDs out first, and *then* return
-        an iterator.
-        """
-        next_day = date + datetime.timedelta(days=1)
-
-        query = {
-            'filter': {
-                'bool': {
-                    'must': [
-                        {
-                            'range': {
-                                'processed_crash.date_processed': {
-                                    'gte': date.isoformat(),
-                                    'lt': next_day.isoformat(),
-                                }
-                            }
-                        },
-                        {
-                            'term': {
-                                'processed_crash.product': product.lower()
-                            }
-                        },
-                        {
-                            'terms': {
-                                'processed_crash.version': [
-                                    x.lower() for x in versions
-                                ]
-                            }
-                        }
-                    ]
-                }
-            }
-        }
-
-        es_index = date.strftime(self.config.elasticsearch.elasticsearch_index)
-        es_doctype = self.config.elasticsearch.elasticsearch_doctype
-
-        crash_ids = []
-        with self.es_context() as es_context:
-            res = helpers.scan(
-                es_context,
-                scroll='2m',  # keep the "scroll" connection open for 2 minutes
-                index=es_index,
-                doc_type=es_doctype,
-                fields=['crash_id'],
-                query=query,
-            )
-            for hit in res:
-                crash_ids.append(hit['fields']['crash_id'][0])
-                if self.config.cap and len(crash_ids) >= self.config.cap:
-                    break
-
-        return iter(crash_ids)
```

Comment:
The problem with these scripts is that they run in a CLI in the infra and we don't really have access to that anymore and thus these aren't helpful.

We have a couple of different scripts that do this sort of thing that are API based so you can run them locally. 

So given that, it's good to remove them.